hujjat.js

```diff
@@ -0,0 +1,7 @@
+const hujjat = {
+    role: "student",
+    profession: "dentistry",
+    interests: "programming and web development",
+    hobbies: ["sports", "hiking"],
+    sisters: 3,
+    brothers: 2
```

Comment:
hey @HujjatGh you are missing the closing curly brace }
can you add it at the end?